jabgui/src/main/java/org/jabref/migrations/PreferencesMigrations.java

```diff
@@ -583,35 +584,59 @@ static void moveApiKeysToKeyring(JabRefCliPreferences preferences) {
      * @param preferences the user's current preferences
      * @implNote The default fields for the "General" tab are defined by {@link FieldFactory#getDefaultGeneralFields()}.
      */
-    static void addICORERankingFieldToGeneralTab(GuiPreferences preferences) {
-        Map<String, Set<Field>> entryEditorPrefs = preferences.getEntryEditorPreferences().getEntryEditorTabs();
-
-        Set<Field> currentGeneralPrefs = entryEditorPrefs.get(Localization.lang("General"));
-        if (currentGeneralPrefs != null) {
-            Set<Field> expectedGeneralPrefs = Set.of(
-                    StandardField.DOI, StandardField.CROSSREF, StandardField.KEYWORDS, StandardField.EPRINT,
-                    StandardField.URL, StandardField.FILE, StandardField.GROUPS, StandardField.OWNER,
-                    StandardField.TIMESTAMP,
-
-                    SpecialField.PRINTED, SpecialField.PRIORITY, SpecialField.QUALITY, SpecialField.RANKING,
-                    SpecialField.READ_STATUS, SpecialField.RELEVANCE
-            );
-            if (!currentGeneralPrefs.equals(expectedGeneralPrefs)) {
-                return;
-            }
+
+    static void migrateGeneralTabDefaultFields(GuiPreferences preferences) {
+        Map<String, Set<Field>> entryEditorPrefs =
+                preferences.getEntryEditorPreferences().getEntryEditorTabs();
+
+        Set<Field> currentGeneralPrefs =
+                entryEditorPrefs.get(Localization.lang("General"));
+
+        if (currentGeneralPrefs == null) {
+            return;
+        }
+
+        // Defaults ICORE (before alpha.3)
```

Comment:
I think, this should be 

```suggestion
        // Defaults fields of tab "General" (before alpha.3)
```